src/main.rs

```diff
@@ -173,6 +165,31 @@ fn run() -> Result<()> {
     Ok(())
 }
 
+/// Initialize the config with the config files and command line options
+/// On UNIX, if local file ('config.toml') does not exists, but the global one does ('/etc/config.toml')
+/// the latter is used.
+/// Otherwise, the local file is used.
+/// Finally, if non of the files could be loaded, the default config is used
+fn initialize_config(cli: ArgMatches<'static>) -> Result<Config> {
+    let config_path = PathBuf::from("config.toml");
+    let global_config_path = PathBuf::from("/etc/config.toml");
```

Comment:
Actually, this was a bug before, it should be `/etc/super-analyzer/config.toml`.